docs/MockFunctionAPI.md

```diff
@@ -594,6 +608,39 @@ test('returns correct data', () => {
 
 Types of classes, functions or objects can be passed as type argument to `jest.Mocked<Source>`. If you prefer to constrain the input type, use: `jest.MockedClass<Source>`, `jest.MockedFunction<Source>` or `jest.MockedObject<Source>`.
 
+### `jest.Replaced<Source>`
+
+The `jest.Replaced<Source>` utility type returns the `Source` type wrapped with type definitions of Jest [replaced property](#replaced-properties).
+
+```ts title="src/utils.ts"
+export function isLocalhost(): boolean {
+  return process.env['HOSTNAME'] === 'localhost';
+}
+```
+
+```ts
+import {afterEach, expect, it, jest} from '@jest/globals';
+import {isLocalhost} from '../utils';
```

Comment:
The problem I see here: it is not clear where this file lives? I mean, if `isLocalHost` is imported from `'../utils'`, where should one place this test file to make it work?

Perhaps, `isLocalHost` function could live in a file with `title="utils.ts"` and test file could have `title="__tests__/utils.test.ts"`?

Reply:
Fixed